cmd/buildah/config.go

```diff
@@ -71,6 +71,11 @@ var (
 			Name:  "workingdir",
 			Usage: "set working `directory` for containers based on image",
 		},
+		cli.StringFlag{
```

Comment:
This flag should be for the buildah command not for the config command, I believe.

Reply:
Should not be hidden, but should be defaulted.  /usr/share/containers/

Reply:
That is what I thought, but I cannot find like a common config struct for buildah - I might be blind.
The only global options are that of the store, which come from storage.
That is why I made the flag for config.
Any ideas on how I should approach this?

Reply:
Global options are in `app.Flags` in `cmd/buildah/main.go`, though I could see this being an option that only matters to `from` and `run`.

Reply:
fixed